sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/Authentication/ContainerRegistryChallengeAuthenticationPolicyTest.cs

```diff
@@ -191,7 +191,7 @@ public async Task ContainerRegistryChallengeAuthenticationPolicy_RefreshesRefres
             int refreshTokenRequests = 0;
             int accessTokenRequests = 0;
             TimeSpan refreshOffset = TimeSpan.FromSeconds(30);
-            TimeSpan expiryTime = TimeSpan.FromSeconds(35);
+            TimeSpan expiryTime = TimeSpan.FromSeconds(32);
```

Comment:
Just curious: why this change?

Reply:
It eliminates a few seconds of time from the test, which gets it under the new global timeout of 5 seconds.

Reply:
Is this going to be flaky? Are we barely under the timeout?

Reply:
It was never flaky in the many times it was run , but since then the timeout has doubled so the risk should be low.